code/game/gamemodes/game_mode.dm

```diff
@@ -674,10 +674,22 @@ proc/display_roundstart_logout_report()
 						continue //Ghosted while alive
 
 	msg += "</span>" // close the span from right at the top
+	return msg
 
-	for(var/mob/M in mob_list)
-		if(M.client && M.client.holder)
-			to_chat(M, msg)
+proc/display_logout_report()
+	var/logout_report = get_logout_report()
+	for(var/s in staff)
+		var/client/C = s
+		if(check_rights(R_MOD|R_ADMIN,0,C.mob))

```

Comment:
Technically like you said in my PR, admins have R_MOD, so you might need to just check that. But I am still worried that devs won't be able to use this proc or verb since I don't think devs have either R_MOD or R_ADMIN. Maybe just make it R_MOD|R_DEV ?

Reply:
I do not see a usecase why a developer would need that information.

Reply:
Me either actuallly